api/rest/place_obs.md

```diff
@@ -136,10 +136,16 @@ If the given node does not have any `Observation`s, then the JSON object would
 be empty.
 
 **NOTE:** Please run `JSON.parse()` on the `payload` field to retrieve the
-compressed data. For example, in JavaScript: `var data =
-JSON.parse(response['payload'])`.
-
-<!--- TODO: provide example to do decompression --->
+compressed data and unzip the data with pako library. For example, in JavaScript:
```

Comment:
nit: "the pako library" (and add a link?)

Reply:
done